calico_node/startup/startup_test.go

```diff
@@ -128,6 +136,13 @@ var _ = Describe("FV tests against a real etcd", func() {
 						Expect(pool.Spec.IPIP.Enabled).To(BeTrue())
 						Expect(pool.Spec.IPIP.Mode).To(Equal(ipip.Mode(expectIpv4IpipMode)))
 					}
+
+					if expectedIPV4NATOutgoing == false {
```

Comment:
I think you should have the same logic as the IPV6 and not check that it `== false` and switch the 2 cases below.